File: src/BreakfastProvider.Api/Models/Responses/MuffinResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record MuffinResponse
{
    public Guid BatchId { get; init; }
    public List<string> Ingredients { get; init; } = [];
    public List<string> Toppings { get; init; } = [];
    public int BakingTemperature { get; init; }
    public int BakingDuration { get; init; }
    public DateTime CreatedAt { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/MuffinResponse.java
package io.lemonlion.breakfast.model.response;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

/** Twin of C# {@code MuffinResponse}. */
public record MuffinResponse(
        UUID batchId,
        List<String> ingredients,
        List<String> toppings,
        int bakingTemperature,
        int bakingDuration,
        Instant createdAt) {
}
